modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/lbs_iterative_methods.h

```diff
@@ -32,16 +26,13 @@ IterativeMethodPETScName(IterativeMethod it_method)
     case IterativeMethod::NONE:
       return "preonly";
     case IterativeMethod::KRYLOV_RICHARDSON:
-    case IterativeMethod::KRYLOV_RICHARDSON_CYCLES:
       return "richardson";
-    case IterativeMethod::GMRES:
-    case IterativeMethod::GMRES_CYCLES:
     case IterativeMethod::KRYLOV_GMRES:
-    case IterativeMethod::KRYLOV_GMRES_CYCLES:
       return "gmres";
     case IterativeMethod::KRYLOV_BICGSTAB:
-    case IterativeMethod::KRYLOV_BICGSTAB_CYCLES:
       return "bcgs";
+    case IterativeMethod::CLASSIC_RICHARDSON:
+      return "";
```

Comment:
I understand this is to return a PETSC iterative method name, but this still looks funny to me. Since the default return value seems to be `""`, is this necessary here?

Reply:
This is solely to prevent a compiler warning that not all values are parsed in the switch.

Reply:
Couldn't you use `default:` here then?